waku/protocol/v2/waku_relay.nim

```diff
@@ -22,9 +22,13 @@ declarePublicGauge total_messages, "number of messages received"
 logScope:
     topic = "WakuSub"
 
-const WakuSubCodec* = "/wakusub/2.0.0-alpha1"
+const WakuSubCodec* = "/vac/waku/relay/2.0.0-alpha2"
 
 type
+  # TODO Decide if this should be WakuSub or WakuRelay
```

Comment:
@kdeme @decanus thoughts here?

Reply:
And why not keep the protocol string also `wakusub`?

What is `WakuSub` used for currently? Is it an abstraction only for the selection of which type of `PubSub` protocol?

Reply:
I almost find WakuRelay more descriptive of what it is than I find WakuSub.

Reply:
@kdeme yes, as per decision to split protocol into three. See https://github.com/vacp2p/specs/pull/170 and https://github.com/vacp2p/specs/issues/169

> What is WakuSub used for currently? Is it an abstraction only for the selection of which type of PubSub protocol?

Correct.

Reply:
> I almost find WakuRelay more descriptive of what it is than I find WakuSub.

I'm fine with either, that's why I added the TODO. If there's no strong consensus now I'd say we leave it as is for now.

Reply:
> I almost find WakuRelay more descriptive of what it is than I find WakuSub.

Alright, changed it. Please review.